resources/kubernetes/operator/helm/values.yaml

```diff
@@ -805,23 +849,23 @@ collectors:
 
 instrumentation:
   name: elastic-instrumentation
-  enabled: true
+  enabled: true # Enable/disable auto-instrumentation.
   exporter:
-    endpoint: http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318
+    endpoint: http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318 # The OpenTelemetry Collector endpoint where telemetry data will be exported.
   propagators:
-    - tracecontext
-    - baggage
-    - b3
+    - tracecontext # W3C TraceContext propagator for distributed tracing.
+    - baggage # Baggage propagator to include baggage information in trace context.
+    - b3 # B3 propagator for Zipkin-based distributed tracing compatibility.
   sampler:
-    type: parentbased_traceidratio
-    argument: "1.0"
+    type: parentbased_traceidratio # Sampler type
+    argument: "1.0" # Sampling rate set to 100% (all traces are sampled).
   java:
     image: docker.elastic.co/observability/elastic-otel-javaagent:1.0.0
   nodejs:
-    image: docker.elastic.co/observability/elastic-otel-node:edge
+    image: docker.elastic.co/observability/elastic-otel-node:0.4.1
```

Comment:
Was this done on purpose?
And why elastic-otel-dotnet:edge still has edge tag?

Reply:
> Was this done on purpose?

Yes, it was initially changed in https://github.com/elastic/opentelemetry/pull/30

>  And why elastic-otel-dotnet:edge still has edge tag?

@jackshirazi @Mpdreamz Could you help with this? Which is the latest available version we should recommend for dotnet autoinstrumentation?

Reply:
A stable elastic-otel-dotnet version has not been released yet, resolving the thread as we will be using "edge" meanwhile.

Thanks for raising this